deploy/rbac-development.json

```diff
@@ -18,7 +18,8 @@
                 "permissions": [
                     {
                         "actions": [
-                            "Microsoft.Resources/subscriptions/resourceGroups/write"
+                            "Microsoft.Resources/subscriptions/resourceGroups/write",
+                            "Microsoft.ContainerRegistry/registries/read"
```

Comment:
TODO: Need dev registry. Fake will do. And need to rbac somewhere.

Reply:
Hi @mjudeikis we should co-ordinate.. for the operator stuff we need to push images so i am moving to arointsvc for dev, but if you need something like arodevsvc then we might as well move straight there?